apps/website/src/pages/docs/carousel.mdx

```diff
@@ -2,76 +2,102 @@
+
+### Carousel.Slider
 
 `Carousel.Slider` is the scrollable list that should consist of `Carousel.Slide` components.
 
-### Slide
+### Carousel.Slide
 
 `Carousel.Slide` is used for the actual slide content. The content can be specified through `children`. It is recommended that the slide content bring its own dimensions (esp. height) and that the dimensions should be the same for all slides.
 
 <PropsTable path={'@itwin/itwinui-react/esm/core/Carousel/CarouselSlide.d.ts'} />
 
-### Navigation
+### Carousel.Navigation
+
+The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
 
-The `Carousel.Navigation` component by default consists of the `PreviousButton` and `NextButton` shown on the left and right, and the `Carousel.DotsList` component shown in the middle. `children` can be specified to override what is shown in this navigation section.
+**Important**: The navigation section must always be present before the slides in DOM order.
 
 <PropsTable path={'@itwin/itwinui-react/esm/core/Carousel/CarouselNavigation.d.ts'} />
 
-### Dots List
+### Carousel.Navigation.PreviousButton
+
+The `Carousel.Navigation.PreviousButton` component facilitates navigation to the previous slide in the carousel.
+
+### Carousel.Navigation.NextButton
+
+The `Carousel.Navigation.NextButton` component enables navigation to the next slide in the carousel.
+
+### Carousel.DotsList
+
+The `Carousel.DotsList` component shows a list of `Carousel.Dot` components which can be used to choose a specific slide.
+
+If used as a descendant of `Carousel`, then this component does not need any props or `children`. `children` can be specified to override the individual dots that are shown. The props can be specified if this component is being used standalone (outside `Carousel`).
 
-The `Carousel.DotsList` component shows a list of `Carousel.Dot` components which can be used to choose a specific slide. If used as a descendant of `Carousel`, then this component does not need any props or `children`. The props can be specified if this component is being used outside `Carousel`. `children` can be specified to override the individual dots that are shown.
+When used standalone, it is important to pay close attention to [accessibility](#accessibility), since you may be missing some parts that make the carousel pattern complete. For this reason, we do not recommend diverging from the standard usage unless you have a very strong need and are confident that you can implement it accessibly.
 
 <PropsTable path={'@itwin/itwinui-react/esm/core/Carousel/CarouselDotsList.d.ts'} />
 
-### Dot
+### Carousel.Dot
 
 `Carousel.Dot` is the actual "dot" component used to activate a slide on clicking. It should be used as a child of `Carousel.Dots`.
```

Comment:
```suggestion
`Carousel.Dot` is the actual "dot" component used to activate a slide on clicking. It should be used as a child of [`Carousel.DotsList`](#carouseldotslist).
```

Reply:
it looks like some of the jsdocs in `CarouselDotsList.tsx` also incorrectly reference `Carousel.Dots` instead of `Carousel.DotsList`. could you update those too?